### PR TITLE
[docs] Voyager docs for v1.3

### PR DESCRIPTION
**Pull request by aishwarya24**
Build preview: https://deploy-preview-17505--infallible-bardeen-164bc9.netlify.app/preview/migrate/

### CODE REVIEW

File: docs/content/preview/migrate/known-issues/mysql.md [hunk 1]
```diff
@@ -501,3 +503,80 @@ CREATE TABLE text_types (
...
+**Example**
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TABLE test(id int, address json);
+ALTER TABLE test ADD CONSTRAINT add_ck CHECK ((json_valid(address)));
+```
+
+The exported schema (failed.sql) is as follows:
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues/mysql.md:526]: Sorry for the confusion. failed.sql is a file created under schema so that the users may directly see the failing DDLs.
We can reword it to `Contents of schema/failed.sql file`

File: docs/content/preview/migrate/known-issues/mysql.md [hunk 2]
```diff
@@ -501,3 +503,80 @@ CREATE TABLE text_types (
...
+
+```sql
+ALTER TABLE test ADD CONSTRAINT add_ck CHECK ((json_valid(address)));
+```
+
+Suggested solution is as follows:
+
+1. Add the following function to the schema or functions or functions.sql file.
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues/mysql.md:534]: `schema/functions/functions.sql` is the file path.

File: docs/content/preview/migrate/known-issues/mysql.md [hunk 3]
```diff
@@ -501,3 +503,80 @@ CREATE TABLE text_types (
...
+    EXCEPTION
+      WHEN OTHERS THEN
+        RETURN false;
+    END;
+    $$
+    LANGUAGE PLPGSQL
+    IMMUTABLE;
+    ```
+
+1. Create the preceding function manually on the target before importing the schema.
```
Comment (shubham-yb) [inline docs/content/preview/migrate/known-issues/mysql.md:552]: Please fix the numbering.

File: docs/content/preview/migrate/reference/yb-voyager-cli.md [hunk 4]
```diff
@@ -84,7 +84,6 @@ The valid *arguments* for export schema are described in the following table:
 | [--source-ssl-root-cert](#ssl-connectivity) <path> | Path to a file containing SSL certificate authority (CA) certificate(s). |
 | [--start-clean](#start-clean) | Clean the schema data directories. |
 | [--use-orafce](#use-orafce) | Use the Orafce extension. Oracle migrations only. |
-| [--verbose](#verbose) | Display extra information in the output. |
```
Comment (shubham-yb) [inline docs/content/preview/migrate/reference/yb-voyager-cli.md:87]: Please remove the verbose flag from `export data` as well. Thanks.

File: docs/content/preview/migrate/reference/yb-voyager-cli.md [hunk 5]
```diff
@@ -198,8 +195,6 @@ The valid *arguments* for export data status are described in the following tabl
 | :------- | :------------------------ |
 | [-e, --export-dir](#export-dir) <path> | Path to the directory where the data files will be exported. |
 | [-h, --help](#command-line-help) | Command line help. |
-| [--send-diagnostics](#send-diagnostics) | Send diagnostics information to Yugabyte. |
-| [--verbose](#verbose) | Display extra information in the output. |
 | [-y, --yes](#yes) | Answer yes to all prompts during the export schema operation. |
```
Comment (shubham-yb) [inline docs/content/preview/migrate/reference/yb-voyager-cli.md:198]: Please remove the `--yes` flag as well. Thanks

File: docs/content/preview/migrate/migrate-steps.md [hunk 6]
```diff
@@ -224,6 +224,18 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
...
 
+#### Accelerate data export (Optional for MySQL/Oracle only)
+
+If you want to speed up the data export for MySQL/Oracle, you can set the environment variable `EXPERIMENTAL_FAST_EXPORT=1` when you run `export data`.
+
+Consider the following caveats before using the feature:
+
+- You need to perform additional steps when you [prepare the source database](#prepare-the-source-database).
+- Some data types are unsupported. For a detailed list, refer to [datatype mappings](../reference/datatype-mapping-mysql/).
+- [--parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument is unsupported.
```
Comment (makalaaneesh) [inline docs/content/preview/migrate/migrate-steps.md:235]: Can you please change it to `parallel-jobs argument will have no effect`

File: docs/content/preview/migrate/install-yb-voyager.md [hunk 7]
```diff
@@ -30,6 +30,10 @@ You can install YugabyteDB Voyager on the following:
 - Disk space must be at least 1.5 times the estimated size of the source database.
```
Comment (ddhodge) [inline docs/content/preview/migrate/install-yb-voyager.md:30]:
```suggestion
- Disk space of at least 1.5 times the estimated size of the source database
```

File: docs/content/preview/migrate/migrate-steps.md [hunk 8]
```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
 
 The options passed to the command are similar to the [`yb-voyager export schema`](#export-schema) command. To export only a subset of the tables, pass a comma-separated list of table names in the `--table-list` argument.
 
+#### Accelerate data export (Optional for MySQL/Oracle only)
```
Comment (ddhodge) [inline docs/content/preview/migrate/migrate-steps.md:227]:
```suggestion
#### Accelerate data export
```

File: docs/content/preview/migrate/migrate-steps.md [hunk 9]
```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
...
+
+If you want to speed up data export for MySQL/Oracle, you can set the environment variable `EXPERIMENTAL_FAST_EXPORT=1` when you run `export data` using yb-voyager.
```
Comment (ddhodge) [inline docs/content/preview/migrate/migrate-steps.md:229]:
```suggestion
For MySQL or Oracle, you can optionally speed up data export by setting the environment variable `EXPERIMENTAL_FAST_EXPORT=1` when you run `export data` using yb-voyager.
```

File: docs/content/preview/migrate/migrate-steps.md [hunk 10]
```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
...
+
+Consider the following caveats before using the feature:
+
+- You need to perform additional steps when you [prepare the source database](#prepare-the-source-database).
+- Some data types are unsupported. For a detailed list, refer to [datatype mappings](../reference/datatype-mapping-mysql/).
+- [--parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument will have no effect.
+- SSL is unsupported.
+- In MySQL RDS, writes are not allowed during the data export process.
+
+{{< note title="Sequence migration considerations" >}}
```
Comment (ddhodge) [inline docs/content/preview/migrate/migrate-steps.md:239]: what's the link? if EXPERIMENTAL_FAST_EXPORT is using sequence migration, that should be worked into/explained in the opening paragraph of this section, rather than being brought up in an otherwise unrelated-looking note at the end of the section

File: docs/content/preview/migrate/mysql.md [hunk 11]
```diff
@@ -26,6 +26,23 @@ Create a database user and provide the user with READ access to all the resource
    GRANT TRIGGER ON source_db_name.* TO 'ybvoyager'@'localhost';
    ```
 
+   Note that if you want to [accelerate data export](#accelerate-data-export-optional-for-mysql-oracle-only), include the following grants additionally as follows:
```
Comment (ddhodge) [inline docs/content/preview/migrate/mysql.md:29]: fix link

File: docs/netlify.toml [hunk 12]
```diff
@@ -578,6 +578,11 @@
   to = "/preview/migrate/:splat"
   force = true
 
+[[redirects]]
+  from = "/v2.16/migrate/*"
```
Comment (ddhodge) [inline docs/netlify.toml:582]: 2.14, 2.12, etc

File: docs/content/preview/migrate/migrate-steps.md (hunk 10 above)
Comment (makalaaneesh) [reply docs/content/preview/migrate/migrate-steps.md:239]: Agreed, it is unrelated. @aishwarya24 can you pls move this to a separate section similar to `Accelerate data export for MySQL or Oracle`

File: docs/content/preview/migrate/migrate-steps.md [hunk 13]
```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
...
+
+Sequence migration consists of two steps: sequence creation and setting resume value (resume value refers to the `NEXTVAL` of a sequence on a source database). A sequence object is generated during export schema and the resume values for sequences are generated during export data. These resume values are then set on the target database just after the data is imported for all tables.
+
+Note that there are some special cases involving sequences such as the following:
+
+- In MySQL, auto-increment column is migrated as a normal column with a sequence attached to it.
+- In PostgreSQL, `SERIAL` datatype and `GENERATED AS IDENTITY` columns use sequence object internally, so resume values for them are also generated during data export.
+- For PostgreSQL and Oracle where sequences are not attached to a column, resume value generation is unsupported with EXPERIMENTAL_FAST_EXPORT=1 environment variable.
```
Comment (makalaaneesh) [inline docs/content/preview/migrate/migrate-steps.md:247]: @aishwarya24 can you pls additionally add this to the list of caveats under the section `Accelerate data export for MySQL or Oracle` ?

File: docs/content/preview/migrate/migrate-steps.md [hunk 14]
```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
 
 The options passed to the command are similar to the [`yb-voyager export schema`](#export-schema) command. To export only a subset of the tables, pass a comma-separated list of table names in the `--table-list` argument.
 
+#### Accelerate data export for MySQL or Oracle
+
+For MySQL or Oracle, you can optionally speed up data export by setting the environment variable `EXPERIMENTAL_FAST_EXPORT=1` when you run `export data` using yb-voyager.
```
Comment (kneeraj) [inline docs/content/preview/migrate/migrate-steps.md:229]: Shouldn't it be `For MySQL and Oracle` instead of `For MySQL or Oracle`

File: docs/content/preview/migrate/performance.md [hunk 15]
```diff
@@ -70,6 +70,8 @@ These performance optimizations apply whether you are importing data using the y
 
 By default, yb-voyager exports four tables at a time. To improve data export, parallelize the export of data from multiple tables using the [–-parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument with the export data command to increase the number of jobs. Setting the value too high can however negatively impact performance; a setting of '4' typically performs well.
 
+If you use EXPERIMENTAL_FAST_EXPORT to [accelerate data export](../migrate-steps/#accelerate-data-export-for-mysql-or-oracle), `--parallel-jobs` will not have any impact on performance and yb-voyager will export only one table at a time.
```
Comment (kneeraj) [inline docs/content/preview/migrate/performance.md:73]: **Note to the developers:**
@sanyamsinghal @makalaaneesh shall we log a WARNING also in the code for this - if this arg is used and the accelerated export is attempted then just show a warning on the screen.

**Review by kneeraj — APPROVED** [review]
Some nits.

**Review by kneeraj — CHANGES_REQUESTED** [review]
Reminder to update what's new section with 1.3 specific enhancements and fixes. @makalaaneesh @sanyamsinghal

File: docs/content/preview/migrate/migrate-steps.md (hunk 10 above)
Comment (makalaaneesh) [reply docs/content/preview/migrate/migrate-steps.md:240]: @aishwarya24 I believe this is still a note. Can you please convert it to a separate section as mentioned? Thanks

File: docs/content/preview/migrate/migrate-steps.md [hunk 16]
```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
...
+For MySQL and Oracle, you can optionally speed up data export by setting the environment variable `EXPERIMENTAL_FAST_EXPORT=1` when you run `export data` using yb-voyager.
+
+Consider the following caveats before using the feature:
+
+- You need to perform additional steps when you [prepare the source database](#prepare-the-source-database).
+- Some data types are unsupported. For a detailed list, refer to [datatype mappings](../reference/datatype-mapping-mysql/).
+- [--parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument will have no effect.
+- SSL is unsupported.
+- In MySQL RDS, writes are not allowed during the data export process.
+- For Oracle where sequences are not attached to a column, resume value generation is unsupported with EXPERIMENTAL_FAST_EXPORT=1 environment variable.
```
Comment (makalaaneesh) [inline docs/content/preview/migrate/migrate-steps.md:238]: Can you pls modify it to `For Oracle where sequences are not attached to a column, resume value generation is unsupported` 

`with EXPERIMENTAL_FAST_EXPORT=1 environment variable` is redundant given that it is under that section.

File: docs/content/preview/migrate/performance.md (hunk 15 above)
Comment (sanyamsinghal) [reply docs/content/preview/migrate/performance.md]: yeah i think it can be helpful also , in case user is not aware of the env var is set and assuming normal export.

File: docs/content/preview/migrate/performance.md [hunk 17]
```diff
@@ -70,6 +70,8 @@ These performance optimizations apply whether you are importing data using the y
 
 By default, yb-voyager exports four tables at a time. To improve data export, parallelize the export of data from multiple tables using the [–-parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument with the export data command to increase the number of jobs. Setting the value too high can however negatively impact performance; a setting of '4' typically performs well.
 
+If you use EXPERIMENTAL_FAST_EXPORT to [accelerate data export](../migrate-steps/#accelerate-data-export-for-mysql-and-oracle), `--parallel-jobs` will not have any impact on performance and yb-voyager will export only one table at a time.
+
 ## Test results
```
Comment (makalaaneesh) [inline docs/content/preview/migrate/performance.md:75]: @aishwarya24 A small restructure required here: 

Please move the `Test Results` section to under [Improve import performance > Techniques to improve performance](https://deploy-preview-17505--infallible-bardeen-164bc9.netlify.app/preview/migrate/performance/#techniques-to-improve-performance) (just above the note) introduction: `Performance test results to demonstrate the above techniques:`

File: docs/content/preview/migrate/reference/yb-voyager-cli.md
Comment (makalaaneesh) [inline docs/content/preview/migrate/reference/yb-voyager-cli.md:1]: Pls add the following arg to `import data file` and to the list of args

--null-string : string that represents null value in the data file (default for csv: ""(empty string), for text: '\N')

File: docs/content/preview/migrate/migrate-steps.md [hunk 18]
```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
 
 The options passed to the command are similar to the [`yb-voyager export schema`](#export-schema) command. To export only a subset of the tables, pass a comma-separated list of table names in the `--table-list` argument.
 
+#### Accelerate data export for MySQL and Oracle
+
+For MySQL and Oracle, you can optionally speed up data export by setting the environment variable `EXPERIMENTAL_FAST_EXPORT=1` when you run `export data` using yb-voyager.
```
Comment (makalaaneesh) [inline docs/content/preview/migrate/migrate-steps.md:229]: @aishwarya24 can you please replace all instances of `EXPERIMENTAL_FAST_EXPORT` to `BETA_FAST_DATA_EXPORT` ? We discussed internally decided that this was more inviting for users to try out.

File: docs/content/preview/migrate/migrate-steps.md [hunk 19]
```diff
@@ -305,7 +329,9 @@ yb-voyager import data file --export-dir <EXPORT_DIR> \
         –-data-dir </path/to/files/dir/> \
         --file-table-map <filename1:table1,filename2:table2> \
         --delimiter <DELIMITER> \
-        –-has-header
+        –-has-header \
+        --null-string
```
Comment (makalaaneesh) [inline docs/content/preview/migrate/migrate-steps.md:333]: `--null-string "<NULL_STRING>"`

File: docs/content/preview/migrate/migrate-steps.md [hunk 20]
```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
 
 The options passed to the command are similar to the [`yb-voyager export schema`](#export-schema) command. To export only a subset of the tables, pass a comma-separated list of table names in the `--table-list` argument.
 
+{{< note title="Sequence migration considerations" >}}
+
+Sequence migration consists of two steps: sequence creation and setting resume value (resume value refers to the `NEXTVAL` of a sequence on a source database). A sequence object is generated during export schema and the resume values for sequences are generated during export data. These resume values are then set on the target database just after the data is imported for all tables.
+
+Note that there are some special cases involving sequences such as the following:
+
+- In MySQL, auto-increment column is migrated as a normal column with a sequence attached to it.
```
Comment (makalaaneesh) [inline docs/content/preview/migrate/migrate-steps.md:233]: `For MySQL, ...`

File: docs/content/preview/migrate/migrate-steps.md [hunk 21]
```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
...
+- In PostgreSQL, `SERIAL` datatype and `GENERATED AS IDENTITY` columns use sequence object internally, so resume values for them are also generated during data export.
```
Comment (makalaaneesh) [inline docs/content/preview/migrate/migrate-steps.md:234]: `For PostgreSQL, ...`

File: docs/content/preview/migrate/performance.md [hunk 22]
```diff
@@ -56,21 +56,13 @@ Use one or more of the following techniques to improve import data performance:
...
-{{< note title="Note" >}}
-
-These performance optimizations apply whether you are importing data using the yb-voyager [import data command](../migrate-steps/#import-data) or the [import data file command](../migrate-steps/#import-data-file).
-
-{{< /note >}}
-
-## Improve export performance
-
-By default, yb-voyager exports four tables at a time. To improve data export, parallelize the export of data from multiple tables using the [–-parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument with the export data command to increase the number of jobs. Setting the value too high can however negatively impact performance; a setting of '4' typically performs well.
+**Test results**
```
Comment (makalaaneesh) [inline docs/content/preview/migrate/performance.md:63]: `Performance Test Results`

File: docs/content/preview/migrate/performance.md [hunk 23]
```diff
@@ -56,21 +56,13 @@ Use one or more of the following techniques to improve import data performance:
 
 - **Add disks** to reduce disk write contention. YugabyteDB servers can be configured with one or multiple disk volumes to store tablet data. If all tablets are writing to a single disk, write contention can slow down the ingestion speed. Configuring the [YB-TServers](../../reference/configuration/yb-tserver/) with multiple disks can reduce disk write contention, thereby increasing throughput. Disks with higher IOPS and better throughput also improve write performance.
 
-- **Enable packed rows** to increase the throughput by more than two times. Enable packed rows on the YugabyteDB cluster by setting the YB-TServer GFlag `ysql_enable_packed_row` to true.
+- **Enable packed rows** (Beta) to increase the throughput by more than two times. Enable packed rows on the YugabyteDB cluster by setting the YB-TServer GFlag `ysql_enable_packed_row` to true.
```
Comment (ddhodge) [inline docs/content/preview/migrate/performance.md:59]: early access +link

Comment (ddhodge) [inline docs/content/preview/migrate/performance.md:59]:
```suggestion
- **Enable packed rows** (Early Access) to increase the throughput by more than two times. Enable packed rows on the YugabyteDB cluster by setting the YB-TServer flag `ysql_enable_packed_row` to true.
```

File: docs/content/preview/migrate/performance.md [hunk 24]
```diff
@@ -56,21 +56,13 @@ Use one or more of the following techniques to improve import data performance:
...
-{{< note title="Note" >}}
-
-These performance optimizations apply whether you are importing data using the yb-voyager [import data command](../migrate-steps/#import-data) or the [import data file command](../migrate-steps/#import-data-file).
-
-{{< /note >}}
-
-## Improve export performance
-
-By default, yb-voyager exports four tables at a time. To improve data export, parallelize the export of data from multiple tables using the [–-parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument with the export data command to increase the number of jobs. Setting the value too high can however negatively impact performance; a setting of '4' typically performs well.
+**Performance test results**
```
Comment (ddhodge) [inline docs/content/preview/migrate/performance.md:63]: why not an h4?

File: docs/content/preview/migrate/performance.md [hunk 25]
```diff
@@ -56,21 +56,13 @@ Use one or more of the following techniques to improve import data performance:
...
 
-## Test results
+Following are the performance test results to demonstrate the preceding techniques:
```
Comment (ddhodge) [inline docs/content/preview/migrate/performance.md:65]:
```suggestion
The following performance test results demonstrate the preceding techniques.
```

File: docs/content/preview/migrate/performance.md [hunk 26]
```diff
@@ -102,6 +94,14 @@ As more optimizations are introduced, average throughput increases. The followin
 
 {{< note title="Note" >}}
 
-With increased optimisations, the CPU usage also shoots up. Permitting the CPU usage beyond 60% may not be advisable as the database requires some cycles to do it's internal operations.
+- With increased optimisations, the CPU usage also shoots up. Permitting the CPU usage beyond 60% may not be advisable as the database requires some cycles to do it's internal operations.
```
Comment (ddhodge) [inline docs/content/preview/migrate/performance.md:97]:
```suggestion
- The more optimizations you use, the more CPU you will require. Allowing CPU use greater than 60% may not be advisable as the database requires some cycles for internal operations.
```

File: docs/content/preview/migrate/performance.md [hunk 27]
```diff
@@ -102,6 +94,14 @@ As more optimizations are introduced, average throughput increases. The followin
...
+
+- These performance optimizations apply whether you are importing data using the yb-voyager [import data command](../migrate-steps/#import-data) or the [import data file command](../migrate-steps/#import-data-file).
 
 {{< /note >}}
+
+## Improve export performance
+
+By default, yb-voyager exports four tables at a time. To improve data export, parallelize the export of data from multiple tables using the [–-parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument with the export data command to increase the number of jobs. Setting the value too high can however negatively impact performance; a setting of '4' typically performs well.
```
Comment (ddhodge) [inline docs/content/preview/migrate/performance.md:105]:
```suggestion
By default, yb-voyager exports four tables at a time. To speed up data export, parallelize the export of data from multiple tables using the [–-parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument with the export data command to increase the number of jobs. Setting the value too high can, however, negatively impact performance; a setting of 4 typically performs well.
```

Comment (ddhodge) [inline docs/content/preview/migrate/performance.md:105]: Not clear if yb-voyager is setting parallel-jobs to 4 by default, or what. number of jobs = number of tables exported in parallel?

File: docs/content/preview/migrate/performance.md [hunk 28]
```diff
@@ -102,6 +94,14 @@ As more optimizations are introduced, average throughput increases. The followin
...
+
+If you use BETA_FAST_DATA_EXPORT to [accelerate data export](../migrate-steps/#accelerate-data-export-for-mysql-and-oracle), `--parallel-jobs` will not have any impact on performance and yb-voyager will export only one table at a time.
```
Comment (ddhodge) [inline docs/content/preview/migrate/performance.md:107]:
```suggestion
If you use BETA_FAST_DATA_EXPORT to [accelerate data export](../migrate-steps/#accelerate-data-export-for-mysql-and-oracle), yb-voyager exports only one table at a time and the `--parallel-jobs` argument is ignored.
```

File: docs/content/preview/migrate/oracle.md [hunk 29]
```diff
@@ -4,6 +4,8 @@ private=true
 +++
 -->
 
+Note that if you want to [accelerate data export](#accelerate-data-export-for-mysql-and-oracle), refer to [Database setup for accelerated data export](#database-setup-for-accelerated-data-export).
```
Comment (ddhodge) [inline docs/content/preview/migrate/oracle.md:7]: second link no work; why link both

File: docs/content/preview/migrate/mysql.md [hunk 30]
```diff
@@ -9,33 +9,50 @@ Create a database user and provide the user with READ access to all the resource
...
+
+   **For MYSQL RDS**
+
+   ```sql
+   GRANT FLUSH_TABLES ON *.* TO 'ybvoyager'@'<client_IP>';
+   GRANT REPLICATION CLIENT ON *.* TO 'ybvoyager'@'<client_IP>';
+   GRANT LOCK TABLES ON <source_db_name>.* TO 'ybvoyager'@'<client_IP>';
    ```
 
 1. If you are running MySQL version >= 8.0.20, grant the global `SHOW_ROUTINE` permission. For older versions, grant the global `SELECT` permission. These permissions are necessary to dump stored procedure/function definitions.
```
Comment (ddhodge) [inline docs/content/preview/migrate/mysql.md:46]:
```suggestion
1. If you are running MySQL version 8.0.20 or later, grant the global `SHOW_ROUTINE` permission. For older versions, grant the global `SELECT` permission. These permissions are necessary to dump stored procedure/function definitions.
```

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/migrate/performance.md (hunk 23 above)
Comment (priyanshi-yb) [reply docs/content/preview/migrate/performance.md:59]: @aishwarya24, yes please link `Enable packed rows (Beta)` with - https://docs.yugabyte.com/preview/reference/configuration/yb-tserver/#packed-row-flags-beta

File: docs/content/preview/migrate/performance.md [hunk 31]
```diff
@@ -56,21 +56,13 @@ Use one or more of the following techniques to improve import data performance:
...
-
-## Improve export performance
-
-By default, yb-voyager exports four tables at a time. To improve data export, parallelize the export of data from multiple tables using the [–-parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument with the export data command to increase the number of jobs. Setting the value too high can however negatively impact performance; a setting of '4' typically performs well.
+**Performance test results**
 
-## Test results
+Following are the performance test results to demonstrate the preceding techniques:
 
 yb-voyager was tested using varying configurations, including more parallel jobs, multiple disks, and a larger cluster. The tests were run using a 28GB CSV file with 350 million rows on YugabyteDB version 2.16.0.0-b90.
```
Comment (priyanshi-yb) [inline docs/content/preview/migrate/performance.md:68]: `import data` command was tested using varying configurations, including more parallel jobs, ....

**Review by makalaaneesh — APPROVED** [review]

**Review by makalaaneesh — APPROVED** [review]

File: docs/content/preview/migrate/release-notes.md [hunk 32]
```diff
@@ -13,6 +13,24 @@ type: docs
 
 Included here are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.3 - May 26, 2023
+
+### Key enhancements
+
+* Export data for MySQL and Oracle is now 2-4x faster. You can set an environment variable `BETA_FAST_DATA_EXPORT=1` to leverage this performance improvement. Most features such as migrating partitioned tables, sequences, and so on are supported in this mode. Refer to the [Export data](../migrate-steps/#export-data) for more details.
+
+* Added support for characters such as backspace(\b) in quote and escape character with `--file-opts` in import data file.
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/migrate/release-notes.md:22]: There were some limitations in older releases - like for example, you couldn't specify \b as the quote character with --file-opts. That has been fixed now.

File: docs/content/preview/migrate/install-yb-voyager.md [hunk 33]
```diff
@@ -27,9 +27,13 @@ You can install YugabyteDB Voyager on the following:
 
 ### Hardware requirements
 
-- Disk space must be at least 1.5 times the estimated size of the source database.
+- Disk space of at least 1.5 times the estimated size of the source database
 - 2 cores minimum (recommended)
 
+### Software requirement
+
+- Java 17 or later
```
Comment (makalaaneesh) [inline docs/content/preview/migrate/install-yb-voyager.md:35]: `Java 11 or later. Recommended Java 17 for better performance`

File: docs/content/preview/migrate/release-notes.md [hunk 34]
```diff
@@ -13,6 +13,24 @@ type: docs
 
 Included here are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.3 - May 26, 2023
+
```
Comment (priyanshi-yb) [inline docs/content/preview/migrate/release-notes.md:17]: @aishwarya24  change the date of the release to `May 30, 2023`

Comment (aishwarya24) [reply docs/content/preview/migrate/release-notes.md:17]: Yes Ill keep this one open and change the date when we are ready to release.

File: docs/content/preview/migrate/performance.md (hunk 27 above)
Comment (aishwarya24) [reply docs/content/preview/migrate/performance.md]: clarified the defaults.

File: docs/content/preview/migrate/release-notes.md (hunk 34 above)
Comment (priyanshi-yb) [reply docs/content/preview/migrate/release-notes.md:17]: Okay, thanks!
